parsers/cxx/cxx_parser_tokenizer.c

```diff
@@ -1088,16 +1140,30 @@ boolean cxxParserParseNextToken(void)
 				))
 			{
 				CXX_DEBUG_PRINT("Ignore token %s",vStringValue(t->pszWord));
-				// FIXME: Handle ignore parens!
+
 				if(szReplacement && *szReplacement)
 				{
+					CXX_DEBUG_PRINT("The token has replacement %s: applying",szReplacement);
 					vStringClear(t->pszWord);
 					vStringCatS(t->pszWord,szReplacement);
+					// FIXME: Do we need to interpret the replacement?
```

Comment:
it probably would be great, but I also think it's not likely to be very important.  But yeah if it could understand replacements to at least keywords like `const`/`extern`/`static` and combination of those, it probably would be awesome for when ignore tokens are really useful.